adcm_client/objects.py

```diff
@@ -661,6 +664,9 @@ def task_list(self, **args) -> "TaskList":
 
     def run(self, **args) -> "Task":
         with allure_step("Run action {}".format(self.name)):
+            config = self._get_config()
```

Comment:
That is incorrect logic. If we have config as parameter, than we have to pass this without interventions. 

Please do the following logic:

* If we have config in args we write this without intervention
* If we have config_diff in args we use setdefault logic over it and pass it to _subcall as config (you need to rename key)